servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRouter.java

```diff
@@ -196,18 +200,25 @@ RouteProviders drainRoutes() {
             return new RouteProviders(allRoutes);
         }
 
+        GrpcExecutionStrategy alreadyRegisteredExecutionStrategy(final String path,
+                                                                 final GrpcExecutionStrategy defaultValue) {
+            return executionStrategies.getOrDefault(path, defaultValue);
+        }
+
         static GrpcRouter.Builder merge(final GrpcRouter.Builder... builders) {
             final Map<String, RouteProvider> routes = new HashMap<>();
             final Map<String, RouteProvider> streamingRoutes = new HashMap<>();
             final Map<String, RouteProvider> blockingRoutes = new HashMap<>();
             final Map<String, RouteProvider> blockingStreamingRoutes = new HashMap<>();
+            final Map<String, GrpcExecutionStrategy> executionStrategies = new HashMap<>();
             for (Builder builder : builders) {
                 mergeRoutes(routes, builder.routes);
                 mergeRoutes(streamingRoutes, builder.streamingRoutes);
                 mergeRoutes(blockingRoutes, builder.blockingRoutes);
                 mergeRoutes(blockingStreamingRoutes, builder.blockingStreamingRoutes);
+                executionStrategies.putAll(executionStrategies);
```

Comment:
```suggestion
                executionStrategies.putAll(builder.executionStrategies);
```

If it is simple, add a test to verify, else it is OK.